src/main/java/com/kodilla/ecommercee/controller/OrderController.java

```diff
@@ -0,0 +1,46 @@
+package com.kodilla.ecommercee.controller;
+
+import com.kodilla.ecommercee.domain.dto.OrderRequestDto;
+import com.kodilla.ecommercee.domain.dto.OrderDto;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+
+@RestController
+@CrossOrigin("*")
+@RequestMapping("/orders")
+public class OrderController {
+    @GetMapping
+    public ResponseEntity<List<OrderDto>> getAllOrders() {
+        List<OrderDto> sampleList = new ArrayList<>();
+        sampleList.add(new OrderDto(1L, 2L, LocalDate.of(2022, 5, 10)));
+        return ResponseEntity.ok(sampleList);
+    }
+
+    @GetMapping(path = "{orderId}")
```

Comment:
In  @DeleteMapping(path = "/{orderId}")  You use / , but not here.

I think (value = "/{userId}") or better only @GetMapping("/{id}") should be better. But your version is OK. Spring is very intelligent and will add / if nessesery.

Reply:
You are right, I'will add slash to be consistent